bootstrap/sql/migrations/native/1.3.1/mysql/schemaChanges.sql

```diff
@@ -0,0 +1,4 @@
+-- Add the supportsProfiler field to the MongoDB connection configuration
+UPDATE dbservice_entity
+SET json = JSON_INSERT(json, '$.connection.config.supportsProfiler', TRUE)
```

Comment:
should this be 1.3.1 though? If we'll deploy all these NoSQL thingies by 1.3.2 - 1.4.0, might be better to leave the 1.3.1 clean

Reply:
moved to 1.4.0